objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -303,8 +303,20 @@ public void run() {
             }
         };
         
-        // Start polling if pollRate is non-negative
-        if (general.get("pollRate") instanceof Integer) {
+        // Start polling if pollTime is non-negative
+        if (general.get("pollTime") instanceof Integer) {
+            polling = (Integer) general.get("pollTime");
```

Comment:
Or put the  "pollRate" check into an else, like you have it, so that you only check it if pollTime is NOT set...
```
polling = -1;
if (general.get("pollTime") instanceof Integer) {
            polling = (Integer) general.get("pollTime");      
} else if (general.get("pollRate") instanceof Integer) {
            polling = (Integer) general.get("pollRate");   // old, deprecated setting
}
if (polling > 0) .....
```